tensorflow/compiler/mlir/hlo/lib/Dialect/mhlo/transforms/lhlo_legalize_to_affine.cc

```diff
@@ -176,6 +176,309 @@ struct ConcatOpConverter : public OpRewritePattern<ConcatenateOp> {
   }
 };
 
+/// Returns a zero value of type `type`. `type` is expected to be either
+/// int or float.
+static Value getZeroValue(Type type, Location loc,
+                          PatternRewriter& rewriter) {
+  assert(eltType.isIntOrFloat() && "Expected int or float");
```

Comment:
eltType is not declared

Reply:
This should have been `type` - I guess the CI runs with assertions disabled.